app/Resources/views/search/layout.html.twig

```diff
@@ -0,0 +1,46 @@
+{% extends 'base.html.twig' %}
+
```

Comment:
I don't want to load my template like this. The controller will load the best suitable for the data to display.